src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_params.py

```diff
@@ -98,10 +99,22 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('hub_name', hub_name_type, options_list=['--name', '-n'], id_part='name')
         c.argument('etag', options_list=['--etag', '-e'], help='Entity Tag (etag) of the object.')
 
-    for subgroup in ['consumer-group', 'policy', 'job', 'certificate']:
+    for subgroup in ['consumer-group', 'policy', 'job', 'certificate', 'routing-endpoint', 'route']:
```

Comment:
what does this 'route' subgroup do here, like a place holder for the implementation for routes?

Reply:
my mistake, I shouldn't include this now because I'll add route commands in this PR later as this is step1 for support routing in IotHub